AlphaWallet/Transfer/ViewControllers/RequestViewController.swift

```diff
@@ -74,7 +78,10 @@ class RequestViewController: UIViewController {
 
 		view.backgroundColor = viewModel.backgroundColor
 		view.addSubview(roundedBackground)
-
+        
+        ensLabel.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(copyEns)))
```

Comment:
Sorry, didn't realise the containers (the rounded rects) are different from the labels. Can you add the gesture recognisers to addressContainerView and ensContainerView instead so the hit test area is bigger?

Reply:
@hboon file was removed, sent changes.